twitter-ads.gemspec

```diff
@@ -27,8 +33,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'multi_json', '~> 1.11'
   s.add_dependency 'oauth', '~> 0.4'
 
-  s.add_development_dependency 'bundler', '~> 1.6'
```

Comment:
This is because Travis CI is now using Bundler 2.x in some cases and we've been seeing `ruby-head` test is always failing. Removing this fixes the issue.